client/my-sites/site-settings/form-analytics-stores.jsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import CompactFormToggle from 'components/forms/form-toggle/compact';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLegend from 'components/forms/form-legend';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+
+class FormAnalyticsStores extends Component {
+	handleToggleChange = ( name ) => () => {
+		this.props.handleToggleChange( name );
+	}
+
+	renderSettings = ( settings, disableAll ) => {
+		const { fields } = this.props;
+
+		return (
+			<div>
+				{
+					settings.map( setting =>
+						<CompactFormToggle
+							checked={ fields.wga ? fields.wga[ setting.key ] : false }
+							disabled={ disableAll }
+							onChange={ this.handleToggleChange( setting.key ) }>
+							{ setting.label }
+						</CompactFormToggle>
+					)
+				}
+			</div>
+		);
+	}
+
+	renderBasicSettings = ( disableAll ) => {
+		const { translate } = this.props;
+		const settings = [
+			{
+				key: 'track_purchases',
+				label: translate( 'Purchase transactions' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **37** times:
`translate( 'Transactions' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).